lib/webpacker/manifest.rb

```diff
@@ -105,11 +105,14 @@ def manifest_type(pack_type)
     def missing_file_from_manifest_error(bundle_name)
       <<-MSG
 Webpacker can't find #{bundle_name} in #{config.public_manifest_path}. Possible causes:
-1. You want to set webpacker.yml value of compile to true for your environment
-   unless you are using the `webpack -w` or the webpack-dev-server.
-2. webpack has not yet re-run to reflect updates.
-3. You have misconfigured Webpacker's config/webpacker.yml file.
-4. Your webpack configuration is not creating a manifest.
+1. You forgot to install node packages (try either `npm install` or `yarn install`)
```

Comment:
does `npm install` work with webpacker?

Reply:
It should but we never mentioned it anywhere so I suggest to remove it here too

Reply:
@jasonfb this is good, but please remove the "npm" part.

@guillaumebriday I'll fix this if we don't hear from @jasonfb. This is a helpful fix.